### PR TITLE
Themes: add light | dark theme

### PR DESCRIPTION
**Pull request by kseniaSs**
Duplicate https://github.com/codex-team/notes.web/pull/16/files
Because 16 pr has problem with git

### CODE REVIEW

File: src/presentation/styles/colors.pcss [hunk 1]
```diff
@@ -32,3 +32,7 @@
   --color-line: var(--color-gray-superlight);
   --color-delimiter-light: var(--color-transparent);
 }
+
+.light {
+  --color-bg: #fff;
```
[the remark this answers is not in the document]
Comment (kseniaSs) [reply src/presentation/styles/colors.pcss:37]: Yes, but it's redefinition for light theme 

Default theme is dark

[the remark this answers is not in the document]
Comment (slaveeks) [reply src/presentation/styles/colors.pcss:37]: let's use `--color-bg: var(--color-white)`

File: src/App.vue [hunk 2]
```diff
@@ -6,8 +6,11 @@
 </template>
 
 <script lang="ts" setup>
+import { useColorMode } from '@vueuse/core'
 import Header from '@/presentation/components/Header.vue';
 import Layout from "@/presentation/layouts/Layout.vue"
+
+const mode = useColorMode()
```
Comment (neSpecc) [inline src/App.vue:13]: seems like `mode` is not used

File: src/presentation/styles/colors.pcss (hunk 1 above)
[the remark this answers is not in the document]
Comment (neSpecc) [reply src/presentation/styles/colors.pcss:37]: You need to redefine all variables (those which are below `Base`)

**Review by slaveeks — APPROVED** [review]

File: src/presentation/styles/colors.pcss [hunk 3]
```diff
@@ -32,3 +33,24 @@
   --color-line: var(--color-gray-superlight);
   --color-delimiter-light: var(--color-transparent);
 }
+
+.light {
+   /**
+   * Text
+   */
+  --color-text-main: var(--color-black);
```
Comment (neSpecc) [inline src/presentation/styles/colors.pcss:41]:
```suggestion
  --color-text-main: var(--color-gray-contrast);
```

File: src/presentation/styles/colors.pcss [hunk 4]
```diff
@@ -32,3 +33,24 @@
...
+  --color-text-heading: var(--color-black);
```
Comment (neSpecc) [inline src/presentation/styles/colors.pcss:42]:
```suggestion
  --color-text-heading: var(--color-gray-contrast);
```

File: src/presentation/styles/colors.pcss [hunk 5]
```diff
@@ -32,3 +33,24 @@
...
+  * Background
+  */
+  --color-bg: var(--color-white);
+  --color-bg-header: var(--color-white);
+  --color-signal: var(--color-blue-bright);
+
+  /**
+  * Line
+  */
+  --color-line: var(--color-gray-light-shade);
```
Comment (neSpecc) [inline src/presentation/styles/colors.pcss:54]:
```suggestion
  --color-line: var(--color-gray-superlight);
```

File: src/presentation/styles/colors.pcss [hunk 6]
```diff
@@ -32,3 +33,24 @@
...
+  --color-delimiter-light: var(--color-gray-light-shade);
```
Comment (neSpecc) [inline src/presentation/styles/colors.pcss:55]:
```suggestion
  --color-delimiter-light: var(--color-gray-superlight);
```

**Review by neSpecc — APPROVED** [review]

**Pull request merged**